module/VuFind/src/VuFind/Config/AccountCapabilities.php

```diff
@@ -177,6 +177,16 @@ public function getSmsSetting()
             ? 'disabled' : 'enabled';
     }
 
+    /**
+     * Get email setting.
+     *
+     * @return bool
+     */
+    public function getEmailSetting(): bool
+    {
+        return !($this->config?->Mail?->require_login ?? true) || $this->getAuth()->loginEnabled();
```

Comment:
Should you use `$this->isAccountAvailable` instead of `$this->getAuth()->loginEnabled` to better cover the privacy special case?

Reply:
I'm not sure about this. Why shouldn't users be able to email records in privacy mode? As far as I know no information needs to be stored in the database for that action.